src/Utilities/Compiler/Options/AnalyzerOptionsExtensions.cs

```diff
@@ -217,5 +218,42 @@ CategorizedAnalyzerConfigOptions ComputeCategorizedAnalyzerConfigOptions()
                 return CategorizedAnalyzerConfigOptions.Empty;
             }
         }
+
+        public static ImmutableArray<(INamedTypeSymbol type, string suffix)> GetAdditionalRequiredSuffixesOption(
```

Comment:
You should be able to replace this method with following once you switch over to using the existing supported type name specification format:

```csharp
public static SymbolNamesOption GetAdditionalRequiredSuffixesOption(
            this AnalyzerOptions options,
            DiagnosticDescriptor rule,
            Compilation compilation,
            CancellationToken cancellationToken)
            => options.GetSymbolNamesOption(EditorConfigOptionNames.AdditionalRequiredSuffixes, namePrefixOpt: "T:", rule, compilation, cancellationToken);

```

Reply:
My suggestion will not work as you need to parse the suffix part specially.